src/azure-web-command-service.ts

```diff
@@ -0,0 +1,77 @@
+import { Guid } from "guid-typescript";
+import { LocalStorageItem } from "./azure-web-local-storage-item";
+
+/**
+ * Manage local stored azure web event commands.
+ */
+export class AzureWebCommandService {
+    private static localStorageKey: string = "SENDMESSAGES";
+
+    /**
+     * Get a stored command by its id.
+     * @param commandId The command id.
+     * @returns If found the command id or null if not found.
+     */
+    public static getCommand(commandId: Guid): LocalStorageItem | null {
+        const storedCommands = this.getStoredCommands();
+        if (storedCommands === null) {
+            return null;
+        }
+
+        const command = storedCommands.find((id) => commandId.toString() === id.correlationId);
+        return command || null;
+    }
+
+    /**
+     * Check if given command is sent.
+     * @param commandId The command id.
+     */
+    public static hasCommand(commandId: Guid): boolean {
+        const storedCommands = this.getCommand(commandId);
+        return storedCommands !== null ? true : false;
```

Comment:
A bit too explicit for my taste. Removing `? true : false` does exactly the same.

Reply:
true :)